sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeed.cs

```diff
@@ -27,46 +29,60 @@ internal DataFeed(DataFeedDetail dataFeedDetail)
         }
 
         /// <summary>
+        /// The unique identifier of this <see cref="DataFeed"/>. Set by the service.
         /// </summary>
         public string Id { get; internal set; }
 
         /// <summary>
+        /// The current ingestion status of this <see cref="DataFeed"/>.
         /// </summary>
         public DataFeedStatus? Status { get; }
 
         /// <summary>
+        /// Date and time(UTC) when this <see cref="DataFeed"/> was created.
         /// </summary>
         public DateTimeOffset? CreatedTime { get; }
 
         /// <summary>
+        /// Whether or not the user who queried the information about this <see cref="DataFeed"/>
```

Comment:
For later discussion: This concept seems strangely named. I would want to call this something like, `IsCurrentUserAdministrator` or `IsRequestingUserAdministrator`, etc.

Reply:
Added to the "design considerations" issue. I wonder why this is needed, since the whole list of administrators is returned.